app/src/main/java/org/stepik/android/view/injection/course/CourseDeeplinkPromoCodeBusModule.kt

```diff
@@ -0,0 +1,16 @@
+package org.stepik.android.view.injection.course
+
+import com.jakewharton.rxrelay2.BehaviorRelay
+import dagger.Module
+import dagger.Provides
+import org.stepic.droid.di.AppSingleton
+import org.stepik.android.domain.course_payments.model.DeeplinkPromoCode
+
+@Module
+object CourseDeeplinkPromoCodeBusModule {
+    @Provides
+    @JvmStatic
+    @AppSingleton
```

Comment:
We can't determine relation course <-> promo code. This will break in case we open several screens with promo code.

Reply:
Will pass the promo code as a parameter in another PR.